mocha.entry.js

```diff
@@ -30,6 +30,8 @@ var webpackConfigBase = {
 describe('[ModernizrWebpackPlugin] Build Tests', function () {
 
   beforeEach(function (done) {
+    this.timeout(5000);
```

Comment:
What's the reasoning behind the timeouts?

Reply:
They were timing out while I was running them on my machine, so I increased it. Maybe it's something to do with the Mac filesystem?

Reply:
I'm going to remove the `timeout()` functions. I was using a docker image for node while testing and it was causing 
 the timeouts.